cmd/appgw-ingress/errors.go

```diff
@@ -0,0 +1,10 @@
+// -------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
```

Comment:
can we move this to `errors` package ?

Reply:
I don' think it's necessary at this point.